core/commands/blame.py

```diff
@@ -353,9 +358,16 @@ def show_commit(self):
         self.view.window().run_command("gs_show_commit", {"commit_hash": commit_hash})
 
     def newst_commit_for_file(self):
```

Comment:
It was used in https://github.com/divmain/GitSavvy/blob/db4711e930825416767536f9ef175574c5d7af61/core/commands/blame.py#L295 but the function has been refactored since then. We could just remove this function from the code base.